hazelcast/src/main/java/com/hazelcast/internal/cluster/MemberInfo.java

```diff
@@ -62,6 +62,11 @@ public MemberInfo(Address address, UUID uuid, Map<String, String> attributes, bo
         this(address, uuid, attributes, liteMember, version, NA_MEMBER_LIST_JOIN_VERSION, addressMap);
     }
 
+    public MemberInfo(Address address, UUID uuid, Map<String, String> attributes, boolean liteMember, MemberVersion version,
+                      boolean isAddressMapExists, Map<EndpointQualifier, Address> addressMap) {
```

Comment:
`isAddressMapExists` is not used in this constructor. is this intentional? A false value would mean an older server version. 
If we don't need the value of this var, it's better to remove this constructor.

Reply:
Yes, it is unused, but AFAICT it can't be avoided, because it is called this way by the generated codec: https://github.com/hazelcast/hazelcast/pull/17003/files#diff-85119ef45bcf87c56a3c4e6485d02c5cR72

I also don't like it though. Do you have any suggestion on how to tweak the code generator?